seqr/management/commands/lift_project_to_hg38_tests.py

```diff
@@ -0,0 +1,249 @@
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_es_variants_for_variant_tuples')
+    def test_command(self, mock_get_es_variants, mock_get_es_samples, mock_logger, mock_input):
+        mock_get_es_samples.return_value = SAMPLE_IDS, INDEX_METADATA
+        es_variants = deepcopy([ES_VARIANT1, ES_VARIANT2])
+        es_variants[0]['xpos'] = 1046394160
+        es_variants[0]['ref'] = 'G'
+        es_variants[0]['alt'] = 'A'
+        es_variants[0]['familyGuids'] = ['F000001_1']
+        es_variants[1]['xpos'] = 1248203925
+        es_variants[1]['ref'] = 'TC'
+        es_variants[1]['alt'] = 'T'
+        mock_get_es_variants.return_value = es_variants
+        mock_input.return_value = 'y'
+        call_command('lift_project_to_hg38', u'--project={}'.format(PROJECT_NAME),
+                     '--es-index={}'.format(ELASTICSEARCH_INDEX))
+
+        calls = [
+            mock.call(u'Updating project genome version for {}'.format(PROJECT_NAME)),
+            mock.call('Validating es index test_index'),
+            mock.call('Lifting over 3 variants (skipping 1 that are already lifted)'),
+            mock.call('Successfully lifted over 2 variants'),
+            mock.call('Successfully updated 2 variants'),
+            mock.call('---Done---'),
+            mock.call('Succesfully lifted over 2 variants. Skipped 1 failed variants. Family data not updated for 0 variants')
+        ]
+        mock_logger.info.assert_has_calls(calls)
+
+        mock_get_es_samples.assert_called_with(ELASTICSEARCH_INDEX)
+
+        mock_get_es_variants.assert_called_with(mock.ANY, [(1001627057, u'G', u'C'), (1046394160, u'G', u'A'), (1248203925, u'TC', u'T')])
+
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.logger')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_elasticsearch_index_samples')
+    def test_command_error_unmatched_sample(self, mock_get_es_samples, mock_logger):
+        sample_ids = deepcopy(SAMPLE_IDS)
+        sample_ids.append('ID_NOT_EXIST')
+        mock_get_es_samples.return_value = sample_ids, INDEX_METADATA
+
+        with self.assertRaises(CommandError) as ce:
+            call_command('lift_project_to_hg38', u'--project={}'.format(PROJECT_NAME),
+                    '--es-index={}'.format(ELASTICSEARCH_INDEX))
+
+        self.assertEqual(ce.exception.message, 'Matches not found for ES sample ids: ID_NOT_EXIST.')
+
+        calls = [
+            mock.call(u'Updating project genome version for {}'.format(PROJECT_NAME)),
+            mock.call('Validating es index test_index')
+        ]
+        mock_logger.info.assert_has_calls(calls)
+
+        mock_get_es_samples.assert_called_with(ELASTICSEARCH_INDEX)
+
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.logger')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.get_elasticsearch_index_samples')
+    @mock.patch('seqr.management.commands.lift_project_to_hg38.Individual')
```

Comment:
Never, ever, mock out a django model class. Change the test to work with the data in the test database and instead use a different return value for `mock_get_es_samples`